debian/gen-flat-repo-fixtures.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -eux
+
+SRCDIR=$(readlink -f "$(dirname "$0")")
+TMPDIR=$(mktemp -d)
+OUTPUTDIR=$(realpath "${SRCDIR}/../$1")
+
+trap 'rm -rf "${TMPDIR}"' EXIT
+
+cd "${TMPDIR}"
+(
+  for CTL in "${SRCDIR}"/asgard/*.ctl
+  do
+    equivs-build --arch ppc64 "${CTL}"
+  done
+)
+
+dpkg-scanpackages . | tee ./Packages | gzip > ./Packages.gz
+
+echo 'Suite: mythology
+Codename: ragnarok
+Architectures: ppc64
+Components: asgard' > Release
+
+apt-ftparchive release . >> Release
+
+mkdir -p "${OUTPUTDIR}"/nest/fjalar/
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/. -t "${OUTPUTDIR}"
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/. -t "${OUTPUTDIR}"/nest/fjalar/
```

Comment:
@quba42 is this what you meant?
I understood, that you wanted the packages in the same directoy, but the `Packages` and `Release` files in two levels down. (i don't see a new testcase otherwise)

Reply:
You're right of course. Thanks. Fixed

Reply:
Oh, i think you need to regenerate the metadata to include the correct relative paths to the packages.
Again. I don't fully understand this repository format. So it would be good, if you could at the very least say that you were able to use apt to retrieve a package from that container. (You know how to start it locally, right?)

Reply:
@mdwllweg trying to locally consume the repo using APT as a test of the repo is a good idea.

On your other point, I am fairly certain it is enough to simply copy `Packages*` and `Release` to the secondary location.
The paths from `Release` to `Packages` are relative to the `Release` file, and so do not change if both are moved.
The paths within `Packages` are relative to the repository root, and so do not change.

Reply:
I just tested the repo locally, and got as far as:
```
package architecture (ppc64) does not match system (amd64)
```
during package install, so I would say the repo principally works for APT.
I tested this via both sets of metadata separately.